app/assets/stylesheets/views/_video.scss

```diff
@@ -9,3 +10,10 @@
     margin-bottom: .1em;
   }
 }
+
+.video-container {
```

Comment:
`.video-container` is used around all of our embedded videos. 

Maybe for now, rename yours `.video-container-flex`? 
Maybe they're converge in the future.